package.json

```diff
@@ -14,6 +14,9 @@
   },
   "dependencies": {
     "@auth/firebase-adapter": "^1.0.0",
+    "@fortawesome/fontawesome-svg-core": "^6.4.0",
```

Comment:
Don't install these packages, use react icons, https://react-icons.github.io/react-icons/icons?name=fa

Reply:
took that line out, used react icon